daemon/clients/pods.py

```diff
@@ -18,22 +18,26 @@ class AsyncPodClient(AsyncPeaClient):
 
     @if_alive
     async def update(
-        self, id: Union[str, 'DaemonID'], dump_path: str, *args, **kwargs
+        self, id: Union[str, 'DaemonID'], dump_path: str, *, uses_with: Dict
     ) -> str:
         """Update a Flow on remote JinaD (only rolling_update supported)
 
         :param id: flow id
         :param dump_path: path of dump from other flow
-        :param args: positional args
-        :param kwargs: keyword args
+        :param uses_with: the uses_with to update the Executor
         :return: flow id
         """
+        if dump_path is not None:
+            if uses_with is not None:
+                uses_with['dump_path'] = dump_path
+            else:
+                uses_with = {'dump_path': dump_path}
```

Comment:
This seems to be repeated many times. Should we add a helper function?

Reply:
we can do in anothe PR